pkgs/development/python-modules/aioitertools/default.nix

```
@@ -0,0 +1,32 @@
+{ buildPythonPackage
+, fetchPypi
+, pythonOlder
+, typing-extensions
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "aioitertools";
+  version = "0.7.0";
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0xw1gg2c6zpw9s7i7q34qf0qxqdfj8nc2k1xnzqpw5q315db071l";
+  };
+
+  propagatedBuildInputs = [
```

according to https://github.com/omnilib/aioitertools/blob/main/pyproject.toml
```suggestion
  propagatedBuildInputs = lib.optionals (pythonAtLeast "3.7") [
```